### PR TITLE
-updated content page of start page

### PR DESCRIPTION
**Pull request by hemerem-ch**
-updated link

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/index/home.njk [hunk 1]
```diff
@@ -18,23 +19,18 @@
...
-      <p class="govuk-body">
-        {{ i18n.youMustHave }}
-        <a href="https://www.gov.uk/guidance/companies-house-identity-verification-standard" data-event-id="Click guidance link">{{ i18n.companiesHouseIdentity }}</a>
-        {{ i18n.outLines }}
-      </p>
-
-      <ul class="govuk-list govuk-list--bullet">
+    <p class="govuk-body">
+      {{ i18n.youMustHave }}
+      <a href="Click guidance link">{{ i18n.companiesHouseIdentity }}</a>
```
Comment (ttingle-ch) [inline src/views/index/home.njk:25]: Is the href correct for this link?

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by ttingle-ch — APPROVED** [review]
LGTM

**Pull request merged**